client/blocks/importer/wordpress/upgrade-plan/upgrade-plan-details.tsx

```diff
@@ -68,14 +67,9 @@ export const UpgradePlanDetails = ( props: Props ) => {
 				</ButtonGroup>
 			</div>
 
-			<div
-				className={ classnames( 'import__upgrade-plan-container', {
-					'feature-list-expanded': showFeatures,
```

Comment:
The 2 classes were removed because after the clean up they weren't needed anymore.